src/components/header/HeaderComponent.vue

```diff
@@ -0,0 +1,63 @@
+<script setup>
+import { ref } from 'vue';
+import { RouterLink } from 'vue-router';
+
+import jazzHeader from '@/assets/images/Jazzheader.png';
+import danceHeader from '@/assets/images/Danceheader.png';
+import yummyHeader from '@/assets/images/Yummyheader.png';
+import homeHeader from '@/assets/images/Homeheader.png';
+import historyHeader from '@/assets/images/Historyheader.png';
+import shopHeader from '@/assets/images/Shopheader.png';
+
+import userHeader from '@/assets/images/Userheader.png';
+import languageHeader from '@/assets/images/Languageheader.png';
+
+const showUserDropdown = ref(false);
+</script>
+
+<template>
+  <body>
+    <div id="app">
+      <header class="header">
+        <nav>
+          <div class="links-container">
+            <div class="user-container">
+              <RouterLink to="/user" class="header-link user" @click.prevent="showUserDropdown = !showUserDropdown">
+                <img :src="userHeader" alt="User" class="header-image small" />
+              </RouterLink>
+              <div v-if="showUserDropdown" class="user-dropdown">
+                <RouterLink to="/login">Login</RouterLink>
+                <RouterLink to="/register">Register</RouterLink>
+              </div>
+            </div>
+            <RouterLink to="/dance" class="header-link dance">
+              <img :src="danceHeader" alt="Dance" class="header-image" />
+            </RouterLink>
+            <RouterLink to="/jazz" class="header-link jazz">
+              <img :src="jazzHeader" alt="Jazz" class="header-image" />
+            </RouterLink>
+            <RouterLink to="/yummy" class="header-link yummy">
+              <img :src="yummyHeader" alt="Yummy" class="header-image" />
```

Comment:
je hebt nu steeds ongebruikte classes zoals yummy en nog een img in de routerlink, ik vind het netter als je de images als een background zet in de css class

Reply:
dan kan je ook makkelijker met text erboven op